slides/lesson1/solution/main_hard_code.tf

```diff
@@ -3,8 +3,10 @@ provider "aws" {
   region = "us-east-2"
 }
 
-
+# Configure default vpc present in
+# AWS account
 resource "aws_default_vpc" "default" {
+  # Adds this tags to the default VPC
```

Comment:
The `tags` property seems self-explanatory, what are your thoughts on removing this comment?